torchrl/data/utils.py

```diff
@@ -42,6 +43,36 @@
 INDEX_TYPING = Union[None, int, slice, str, Tensor, List[Any], Tuple[Any, ...]]
 
 
+def dense_stack_tds(
+    td_list: Union[typing.Sequence[TensorDictBase], LazyStackedTensorDict],
+    stack_dim: int = None,
+) -> TensorDictBase:
+    """Densely stack a list of TensorDictBase objects (or a LazyStackedTensorDict) given that they have the same structure.
+
+    This must be used when some of the tds involved can have LazyTds among keys (or keys of keys, recursively).
```

Comment:
let's avoid abbreviations in docstrings.
Even better: we can link to tensordict classes by using 
```
:class:`tensordict.stuff`
```